docs/src/main/paradox/features/templating.md

```diff
@@ -1,21 +1,32 @@
   [st]: http://www.stringtemplate.org/
 
 Templating
-----------
+==========
 
-Paradox uses [StringTemplate][st] for the basic templating. For example:
+Paradox uses [StringTemplate][st] for the basic templating.
 
-```
-$page.title$
-```
+Here is a list of supported properties that can be used in the string template files:
 
-is substituted with the title of the page.
+- `$page.title$` : display the title of the page, which corresponds to the highest level of title on the page. More info about use of titles @ref[here](organizing-pages.md#titles)
+- `$page.content$` : display the processed content of your entire markdown source file.
+- `$page.base$` : display the relative path to the root folder of the project.
+- `$page.home$`, `$page.prev$`, `$page.next$` : gives references to specified "links". `home` corresponds to the root page of the project, `next` is the next page, and `prev` the previous one in the paradox navigation. 
+  Each "link" can provide four fields:
+  - `href` : displays the relative path.
+  - `html` : gives directly the url leading to this link.
+  - `title` : displays the title of the link (same process than `$page.title$`)
+  	- `active` : determines if the link is currently active or not. Particularly useful if we want to check whether we are on the main page or not: `$if(page.home.active)$`
+- `$page.breadcrumbs$` : gives the list of the parent folder of the current page and each parent is displayed as a link.
+- `$page.navigation$` : gives the links of all pages in the project (except the main page).
+- `$page.subheaders$` : determines if the current page contains subheaders or not, which concretely means that it contains multiple sections depending on the `toc-.
+- `$page.toc$` : in relation with `$page.subheaders$`, displays the list of all sections on the page as anchor links.
+- `$page.properties$`: displays purely the list of the properties for the current page, which contains the actual properties of the page and the global ones shared to all pages. To add properties at page level and how to use them, look at [this](#delimiters).
 
```

Comment:
### custom properties

- In markdown file use `@props("foo")`
- In templates use  `$page.properties.("foo")$`
- to add custom properties to the entire document, add `paradoxProperties += ("some.version" -> "0.1.0")` to build.sbt
- to add to custom properties to one page only, use properties front matter.